### PR TITLE
Feature/sgar 126 make place of birth optional

### PR DESCRIPTION
**Pull request by anushasivakumar**
What changes does this PR bring?
Changes to 'Add a person' form - Making Place of birth optional and moving it below Gender
Place of birth will be optional in both the GAR online form and the uploaded GAR file

Definition of Done
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.

[ x] Have you built the code locally and confirmed its working?
[ x] Is the build confirmed to be passing on CI?
[ x] Have you added enough relevant unit tests for your change?
[] Have you added enough relevant E2E tests for your change?
[ x] Have you updated any relevant documentation as part of this change?
[ x] Has this change been tested against the Acceptance Criteria?
[ x] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

**Comment by sjose012** [comment]
I think there is one issue while trying to submit a GAR with this change
steps:
1. Add a new person with no value in Place of birth
2. Save that person
3. Create a GAR with same saved person
4. While clicking 'Continue' after adding the above person to GAR, the error shown in the image is displayed 
5. console shows below error
node        | debug: common/config/endpoints.js: Calling get GAR people endpoint http://api:5000/v0.2.0/gar/bddd8f72-c112-4b3d-ad4a-4c9168b7bd6d/people?page=1&per_page=10000
api         | 172.18.0.6 - - [12/Apr/2023 08:05:34] "GET /v0.2.0/gar/bddd8f72-c112-4b3d-ad4a-4c9168b7bd6d/people?page=1&per_page=10000 HTTP/1.1" 200 -
node        | debug: common/services/garApi.js: Succ
essfully called GAR people endpoint
node        | info: app/garfile/manifest/post.controller.js: Manifest validation failed, redirecting with error msg
Screenshot 2023-04-12 at 09 07 16
Screenshot 2023-04-12 at 09 12 59

File: src/test/validators.test.js [hunk 1]
```diff
@@ -383,6 +423,41 @@ describe('Validator', () => {
...
+    expect(validator.isValidOptionalStringLength(emptyString)).to.be.true;
+  });
+
+  it('Should return false when an optional string has only Symbols', () => {
+    const onlySymbols = '$$' ;
+    expect(validator.isValidOptionalStringLength(onlySymbols)).to.be.false;
+  });
+
+  it('Should return false when an optional string has leading spaces', () => {
+    const onlySymbols = ' London' ;
```
[the remark this answers is not in the document]
Comment (anushasivakumar) [reply src/test/validators.test.js]: Addressed this in a new commit, kindly verify

**Comment by anushasivakumar** [comment]
> I think there is one issue while trying to submit a GAR with this change steps:
> 
> 1. Add a new person with no value in Place of birth
> 2. Save that person
> 3. Create a GAR with same saved person
> 4. While clicking 'Continue' after adding the above person to GAR, the error shown in the image is displayed
> 5. console shows below error
>    node        | debug: common/config/endpoints.js: Calling get GAR people endpoint http://api:5000/v0.2.0/gar/bddd8f72-c112-4b3d-ad4a-4c9168b7bd6d/people?page=1&per_page=10000
>    api         | 172.18.0.6 - - [12/Apr/2023 08:05:34] "GET /v0.2.0/gar/bddd8f72-c112-4b3d-ad4a-4c9168b7bd6d/people?page=1&per_page=10000 HTTP/1.1" 200 -
>    node        | debug: common/services/garApi.js: Succ
>    essfully called GAR people endpoint
>    node        | info: app/garfile/manifest/post.controller.js: Manifest validation failed, redirecting with error msg
> 
> Screenshot 2023-04-12 at 09 07 16
> 
> Screenshot 2023-04-12 at 09 12 59

Addressed this in a new commit, kindly verify @sjose012

**Review by sjose012 — APPROVED** [review]
Look good.

**Review by ho-henry-senior — APPROVED** [review]

**Pull request merged**